integrations-and-supported-tools/pytorch/scripts/helpers.py

```diff
@@ -0,0 +1,46 @@
+import torch
```

Comment:
we should bring those back into the main file -> again sorry for the confusion. 
remember that those examples should explain what neptune support for pytorch with as little (or as much) complexity as needed.

Reply:
Actually I think the helpers make it work. 

What I will do is make some changes. Make sure all logging is explicit in the main file.

Reply:
awesome